go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -30,13 +31,20 @@ import (
 const KindSparkApplication = "SparkApplication"
 const sparkDriverUI = "sparkDriverUI"
 const sparkHistoryUI = "sparkHistoryUI"
+const sparkExecutorKey = "spark.executor.instances"
 
 var sparkTaskType = "spark"
 
 // Spark-specific configs
 type Config struct {
 	DefaultSparkConfig    map[string]string `json:"spark-config-default" pflag:",Key value pairs of default spark configuration that should be applied to every SparkJob"`
 	SparkHistoryServerURL string            `json:"spark-history-server-url" pflag:",URL for SparkHistory Server that each job will publish the execution history to."`
+	SparkLimits           Limits            `json:"spark-limits" pflag:", Spark Limits that can be set for each job execution."`
+}
+
+// Spark Limits config
+type Limits struct {
+	ExecutorLimit string `json:"executor-limit" pflag:",Max limit on number of executors allowed to be launched."`
```

Comment:
nit: would suggest renaming to ExecutorCountLimit or similar just to be more explicit in-case  if we want to add memory or cpu limits as well.

Reply:
Updated.